app/models/request.rb

```diff
@@ -29,4 +30,8 @@ def generate_request(params)
       course_requests.create(course_id: course_id)
     end
   end
+
+  def send_email
+    AdminMailer.send_request(student, self).deliver_now
```

Comment:
This doesn't work as of now actually. I looked a bit and it doesn't make sense to use `self` here without assigning it some properties as it's done in `admin.rb`, and even then it seems a bit sketch. Probs ask charles to confirm, and also give this a good read: https://www.sitepoint.com/rubys-top-self-object/, should clarify things a lot. Basically tests fail rn since I believe the self object is passed and interpreted as a string, which doesn't have the ".course method'.